app/javascript/src/components/Dashboard/Notes/NewNoteForm.jsx

```diff
@@ -1,36 +1,89 @@
-import React from "react";
+import React, { useState } from "react";
 import * as yup from "yup";
 import { Formik, Form } from "formik";
-import { Input, Textarea } from "neetoui/formik";
-import { Button } from "neetoui";
-import notesApi from "apis/notes";
+import { Input, Textarea, Select } from "neetoui/formik";
+import { Button, DateInput, Switch, Toastr } from "neetoui";
 
 export default function NewNoteForm({ onClose, refetch }) {
-  const handleSubmit = async values => {
+  const [dueDateCheck, setDueDateCheck] = useState(false);
+  const [selectedDueDate, setSelectedDueDate] = useState(new Date());
+
+  const handleSubmit = async () => {
     try {
-      await notesApi.create(values);
       refetch();
       onClose();
+      Toastr.success("New note added successfully");
     } catch (err) {
       logger.error(err);
     }
   };
+
+  const tagOptions = [
+    { label: "Internal", value: "internal" },
+    { label: "External", value: "external" },
+  ];
+
+  const contactOptions = [
+    { label: "Amal Dinesh", value: "amaldinesh" },
+    { label: "Karthik Menon", value: "karthikmenon" },
+  ];
   return (
     <Formik
       initialValues={{
-        title: "",
-        description: "",
+        title: "React Onboarding Challenge",
+        tags: tagOptions[0],
+        description: "Please use neeto-ui and tailwindCSS.",
+        assignedContact: contactOptions[0],
       }}
```

Comment:
Please move `initialValues` and `validationSchema` to a separate constants file.